desktop/release-notes.md

```diff
@@ -23,13 +23,98 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.11.0
+2022-07-28
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added two new extensions ([vcluster](https://hub.docker.com/extensions/loftsh/vcluster-dd-extension) and [PGAdmin4](https://hub.docker.com/extensions/mochoa/pgadmin4-docker-extension)) to the Extensions Marketplace.
+- Ability to sort extensions has been added to the Extensions Marketplace.
+- No more feedback nagging. Thanks to all your feedback, we identified a bug that was causing some users to be asked for feedback too frequently. You should now only be asked for feedback twice a year.
+- Custom theme settings for Docker Desktop allow you to specify dark or light mode for Docker Desktop independent of your device settings. Fixes [docker/for-win#12747](https://github.com/docker/for-win/issues/12747)
+- Add new flag for Windows installer, `--no-windows-containers` to disable Windows containers integration.
+
```

Comment:
- Added a new flag for Mac install command, `--user <username>` that setup Docker Desktop for a specific user and doesn't install privileged daemon nor asks for admin password
(cc @stephen-turner @christophermclellan)